code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -92,6 +92,13 @@
 	walk_to(src,0)
 	last_found = world.time
 
+/mob/living/simple_animal/bot/secbot/electrocute_act()//shocks only make him angry

```

Comment:
Replace the last bit with
`electrocute_act(shock_damage, source, siemens_coeff = 1, safety = 0, override = 0, tesla_shock = 0, illusion = 0, stun = TRUE)`

Reply:
Also https://github.com/tgstation/tgstation/pull/45291 just got merged and might conflict a tiny bit so if it still fails use the args from that pr

Reply:
Give beepsky the shock_immune trait and call parent. That's why I refactored electrocute_act in the first place, so stuff sends the god damn signal.